otter/integration/lib/cloud_load_balancer.py

```diff
@@ -22,6 +22,20 @@
 )
 
 
+def _pending_update_to_transient(f):
+    """
+    A cloud load balancer locks on every update, so to ensure that the test
+    doesn't fail because of that, we want to retry POST/PUT/DELETE commands
+    issued by the test.  This is a utility function that checks if a treq
+    API failure is a 422 PENDING_UDPATE failure, and if so, re-raises a
+    TransientRetryError instead.
+    """
+    if f.check(APIError):
```

Comment:
Non-blocking: You could do `f.trap(APIError)` and need not have `return f` in the end.